samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -127,8 +128,9 @@
    * When sending messages to any other {@code OutputStream<M>}, messages are partitioned using a null partition key.
    *
    * @param outputStream the output stream to send messages to
+   * @return the input {@link MessageStream}
    */
-  void sendTo(OutputStream<M> outputStream);
+  MessageStream<M> sendTo(OutputStream<M> outputStream);
```

Comment:
more documentation please. refer to previous comment on sink api.

Reply:
Seems a little complicated. Not sure what written here refers to `sending to the underlying system or flushing to it`. 

Can we simplify the note along the lines of "no happens before guarantee between the message that is propagated to downstream DAG and the send to the underlying output system". 

WDYT?